From bab79e6b84312f347b6950f52725d7b731a99a93 Mon Sep 17 00:00:00 2001
From: Sheng Nan <b38800@freescale.com>
Date: Wed, 16 Jan 2013 11:06:30 +0800
Subject: ENGR00240228: IPUv3: CSI: Correct enum definition of
 IPU_CSI_DATA_WIDTH

The current definition for with 10 and 16 is not correct.
IPU_CSI_DATA_WIDTH_10 = 2;
IPU_CSI_DATA_WIDTH_16 = 3;

According to the latest i.MX6DQ RM, the correct value should be:
IPU_CSI_DATA_WIDTH_10 = 3;
IPU_CSI_DATA_WIDTH_16 = 9;

Signed-off-by: Sheng Nan <b38800@freescale.com>
---
 arch/arm/plat-mxc/include/mach/ipu-v3.h |   10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/arch/arm/plat-mxc/include/mach/ipu-v3.h b/arch/arm/plat-mxc/include/mach/ipu-v3.h
index 519e628..fb9dc38 100755
--- a/arch/arm/plat-mxc/include/mach/ipu-v3.h
+++ b/arch/arm/plat-mxc/include/mach/ipu-v3.h
@@ -1,6 +1,6 @@
 /*
  * Copyright (c) 2010 Sascha Hauer <s.hauer@pengutronix.de>
- * Copyright (C) 2011-2012 Freescale Semiconductor, Inc.
+ * Copyright (C) 2011-2013 Freescale Semiconductor, Inc.
  *
  * This program is free software; you can redistribute it and/or modify it
  * under the terms of the GNU General Public License as published by the
@@ -419,10 +419,10 @@ typedef struct {
  * Enumeration of CSI data bus widths.
  */
 enum {
-	IPU_CSI_DATA_WIDTH_4,
-	IPU_CSI_DATA_WIDTH_8,
-	IPU_CSI_DATA_WIDTH_10,
-	IPU_CSI_DATA_WIDTH_16,
+	IPU_CSI_DATA_WIDTH_4 = 0,
+	IPU_CSI_DATA_WIDTH_8 = 1,
+	IPU_CSI_DATA_WIDTH_10 = 3,
+	IPU_CSI_DATA_WIDTH_16 = 9,
 };
 
 /*!
-- 
1.7.9.5

